scss/theme/stanford-news.scss

```diff
@@ -28,25 +28,32 @@
 // Title field.
 
 h1 {
-  font-size: 42px;
   color: color(text-active);
+  font-size: 42px;
 }
 // Author field.
 
 .field-s-news-author {
-  text-transform: uppercase;
-  padding-top: 10px;
   padding-bottom: 10px;
+  padding-top: 10px;
+  text-transform: uppercase;
   @include grid-media($media-lg) {
     padding-left: 110px;
   }
 }
+
+.section-news {
+  .field-s-news-author {
+    display: block;
+  }
+}
+
```

Comment:
@cjwest do we want to put this together with the other author field stuff? Maybe it all should be nested inside of section-news?

Reply:
Hi @buttonwillowsix ,

That was my thought originally. However, I decided against it because I don't know where all these styles are getting applied. If we nest it in `section-news` the styles might not apply in some situations. It might be better to drop the `section-news` altogether. What do you think?

Reply:
@cjwest I checked and the author stuff IS only here. So, yes, let's drop the section-news. That was me being overly cautious.